### DISCUSSION TITLE
Customisation of Dendron Native Preview

### DISCUSSION THREAD

**Discussion by agichim** [dendronhq/dendron#1182] [RFCs/Ideas]
Hi, here's this discussion getting started on future features for Dendron's native preview. Aside from the most common CSS styling, I'll list below 2 or 3 things for now.

- ability to toggle on/off the preview of `title` and `description` of the frontmatter. 
- exporting options as the ones that come with Markdown Preview Enhanced (this is not super necessary really, I'm using Zettlr to view the same directory using Zettlr features when needed and it has good .pdf export for example).

**Comment by kevinslin** [comment]
To add some context, the native preview uses ant design as our frontend framework. 

The plan is to let users override [all ant selectors](https://ant.design/docs/react/customize-theme) with their own values. We will probably also look to introduce specific dendron selectors as makes sense.

**Reply by kevinslin** [reply]
Note that the same customization for the preview will also be ported to Dendron's NextJS based publishing

**Comment by ScriptAutomate** [comment]
Related: https://github.com/dendronhq/dendron/discussions/1298
